verifiers/internal/gha/provenance_test.go

```diff
@@ -1182,3 +1185,111 @@ func Test_VerifyVersionedTag(t *testing.T) {
JzIiwic3Vic2NyaXB0aW9uX3VybCI6Imh0dHBzOi8vYXBpLmdpdGh1Yi5jb20vcmVwb3MvZW50ZXJhZ2E2L3Nsc2EtbHZsMy1nZW5lcmljLXByb3ZlbmFuY2Utd2l0aC1iYXplbC1leGFtcGxlL3N1YnNjcmlwdGlvbiIsInN2bl91cmwiOiJodHRwczovL2dpdGh1Yi5jb20vZW50ZXJhZ2E2L3Nsc2EtbHZsMy1nZW5lcmljLXByb3ZlbmFuY2Utd2l0aC1iYXplbC1leGFtcGxlIiwidGFnc191cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3JlcG9zL2VudGVyYWdhNi9zbHNhLWx2bDMtZ2VuZXJpYy1wcm92ZW5hbmNlLXdpdGgtYmF6ZWwtZXhhbXBsZS90YWdzIiwidGVhbXNfdXJsIjoiaHR0cHM6Ly9hcGkuZ2l0aHViLmNvbS9yZXBvcy9lbnRlcmFnYTYvc2xzYS1sdmwzLWdlbmVyaWMtcHJvdmVuYW5jZS13aXRoLWJhemVsLWV4YW1wbGUvdGVhbXMiLCJ0b3BpY3MiOltdLCJ0cmVlc191cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3JlcG9zL2VudGVyYWdhNi9zbHNhLWx2bDMtZ2VuZXJpYy1wcm92ZW5hbmNlLXdpdGgtYmF6ZWwtZXhhbXBsZS9naXQvdHJlZXN7L3NoYX0iLCJ1cGRhdGVkX2F0IjoiMjAyMy0wNS0xOVQwMToxOTozOVoiLCJ1cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3JlcG9zL2VudGVyYWdhNi9zbHNhLWx2bDMtZ2VuZXJpYy1wcm92ZW5hbmNlLXdpdGgtYmF6ZWwtZXhhbXBsZSIsInZpc2liaWxpdHkiOiJwdWJsaWMiLCJ3YXRjaGVycyI6MCwid2F0Y2hlcnNfY291bnQiOjAsIndlYl9jb21taXRfc2lnbm9mZl9yZXF1aXJlZCI6ZmFsc2V9LCJzZW5kZXIiOnsiYXZhdGFyX3VybCI6Imh0dHBzOi8vYXZhdGFycy5naXRodWJ1c2VyY29udGVudC5jb20vdS83ODk1MzYwND92PTQiLCJldmVudHNfdXJsIjoiaHR0cHM6Ly9hcGkuZ2l0aHViLmNvbS91c2Vycy9lbnRlcmFnYTYvZXZlbnRzey9wcml2YWN5fSIsImZvbGxvd2Vyc191cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3VzZXJzL2VudGVyYWdhNi9mb2xsb3dlcnMiLCJmb2xsb3dpbmdfdXJsIjoiaHR0cHM6Ly9hcGkuZ2l0aHViLmNvbS91c2Vycy9lbnRlcmFnYTYvZm9sbG93aW5ney9vdGhlcl91c2VyfSIsImdpc3RzX3VybCI6Imh0dHBzOi8vYXBpLmdpdGh1Yi5jb20vdXNlcnMvZW50ZXJhZ2E2L2dpc3Rzey9naXN0X2lkfSIsImdyYXZhdGFyX2lkIjoiIiwiaHRtbF91cmwiOiJodHRwczovL2dpdGh1Yi5jb20vZW50ZXJhZ2E2IiwiaWQiOjc4OTUzNjA0LCJsb2dpbiI6ImVudGVyYWdhNiIsIm5vZGVfaWQiOiJNRFE2VlhObGNqYzRPVFV6TmpBMCIsIm9yZ2FuaXphdGlvbnNfdXJsIjoiaHR0cHM6Ly9hcGkuZ2l0aHViLmNvbS91c2Vycy9lbnRlcmFnYTYvb3JncyIsInJlY2VpdmVkX2V2ZW50c191cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3VzZXJzL2VudGVyYWdhNi9yZWNlaXZlZF9ldmVudHMiLCJyZXBvc191cmwiOiJodHRwczovL2FwaS5naXRodWIuY29tL3VzZXJzL2VudGVyYWdhNi9yZXBvcyIsInNpdGVfYWRtaW4iOmZhbHNlLCJzdGFycmVkX3VybCI6Imh0dHBzOi8vYXBpLmdpdGh1Yi5jb20vdXNlcnMvZW50ZXJhZ2E2L3N0YXJyZWR7L293bmVyfXsvcmVwb30iLCJzdWJzY3JpcHRpb25zX3VybCI6Imh0dHBzOi8vYXBpLmdpdGh1Yi5jb20vdXNlcnMvZW50ZXJhZ2E2L3N1YnNjcmlwdGlvbnMiLCJ0eXBlIjoiVXNlciIsInVybCI6Imh0dHBzOi8vYXBpLmdpdGh1Yi5jb20vdXNlcnMvZW50ZXJhZ2E2In0sIndvcmtmbG93IjoiLmdpdGh1Yi93b3JrZmxvd3MvdGVzdC12ZXJpZmllci55YW1sIn19LCJyZXNvbHZlZERlcGVuZGVuY2llcyI6W3sidXJpIjoiZ2l0K2h0dHBzOi8vZ2l0aHViLmNvbS9lbnRlcmFnYTYvc2xzYS1sdmwzLWdlbmVyaWMtcHJvdmVuYW5jZS13aXRoLWJhemVsLWV4YW1wbGVAcmVmcy9oZWFkcy9tYWluIiwiZGlnZXN0Ijp7ImdpdENvbW1pdCI6ImFiOWE0NThmNjU5ODI2MjVkZDgzODg0NDIzZmE0NTlhMjI2NGU0MjkifX1dfSwicnVuRGV0YWlscyI6eyJidWlsZGVyIjp7ImlkIjoiaHR0cHM6Ly9naXRodWIuY29tL3Nsc2EtZnJhbWV3b3JrL3Nsc2EtZ2l0aHViLWdlbmVyYXRvci8uZ2l0aHViL3dvcmtmbG93cy9idWlsZGVyX2JhemVsX3Nsc2EzLnltbEByZWZzL3RhZ3MvdjEuOC4wIn0sIm1ldGFkYXRhIjp7Imludm9jYXRpb25JZCI6Imh0dHBzOi8vZ2l0aHViLmNvbS9lbnRlcmFnYTYvc2xzYS1sdmwzLWdlbmVyaWMtcHJvdmVuYW5jZS13aXRoLWJhemVsLWV4YW1wbGUvYWN0aW9ucy9ydW5zLzU3ODgzNDIzODEvYXR0ZW1wdHMvMSJ9fX19",
```

Comment:
Can you put these large contents in a file as DSSE format (under the existing testdata folder). Same comment for the  other test.

Reply:
@laurentsimon Restructured tests!